apps/system/js/utility_tray.js

```diff
@@ -236,7 +259,15 @@ window.UtilityTray = {
           evt.preventDefault();
         }
 
+        var oldDelta = this.lastDelta;
+
         this.onTouchMove(evt.touches[0]);
+
+        var move = this.lastDelta - oldDelta;
+        if (Math.abs(move) > 0) {
```

Comment:
What about in the case where we are touch moving with a constant velocity? As unlikely as that seems, if we had two identical delta Ys in a row we wouldn't count that as a move, and I'm not sure that is what we want here?